### PR TITLE
Fix text overlapping clear button in our custom combobox

### LINKED ISSUE #2601: The delete icon in dropdowns overlaps long texts
Only noticed it on Packets Sizes, but probably in theory the same for all others with long input.

### PR DESCRIPTION
**Pull request by floreks**
**Which issue(s) this PR fixes**:
Fixes #2601

Zrzut ekranu z 2020-09-18 14-06-49

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: src/app/shared/components/combobox/style.scss [hunk 1]
```diff
@@ -16,3 +16,18 @@ input {
   outline: none;
   padding: 16px 0 16px 16px;
 }
+
+mat-select {
```
Comment (maciaszczykm) [inline src/app/shared/components/combobox/style.scss:20]: Doesn't it happen anywhere else that we have close icon inside mat-select? I'm asking as this one will fix it only for combobox component.

Comment (floreks) [reply src/app/shared/components/combobox/style.scss:20]: Good point. I'll add a global class.

**Pull request merged**